### ISSUE TITLE
Errors during plugins:migrate on mysql

### ISSUE THREAD

**Issue by VoronyukM** [go2null/redmine_custom_workflows#5]
At the system:
```
openSUSE 13.1 (Bottle) (x86_64)
mysql  Ver 15.1 Distrib 5.5.33-MariaDB, for Linux (x86_64) using readline 5.1
redmine 2.5.1
```
there is a couple of errors like this:
```
> rake redmine:plugins:migrate RAILS_ENV=development
... (all is ok) ...
==  ChangeCustomWorkflowsDescriptionType: migrating ===========================
-- change_column(:custom_workflows, :description, :text, {:null=>false, :default=>""})
rake aborted!
StandardError: An error has occurred, all later migrations canceled:

Mysql2::Error: BLOB/TEXT column 'description' can't have a default value: ALTER TABLE `custom_workflows` CHANGE `description` `description` text DEFAULT '' NOT NULL/home/redmine/redmine-test/plugins/redmine_custom_workflows/db/migrate/20120628060102_change_custom_workflows_description_type.rb:3:in `up'
/home/redmine/redmine-test/lib/redmine/plugin.rb:452:in `migrate_plugin'
/home/redmine/redmine-test/lib/redmine/plugin.rb:424:in `migrate'
/home/redmine/redmine-test/lib/redmine/plugin.rb:438:in `block in migrate'
/home/redmine/redmine-test/lib/redmine/plugin.rb:437:in `each'
/home/redmine/redmine-test/lib/redmine/plugin.rb:437:in `migrate'
/home/redmine/redmine-test/lib/tasks/redmine.rake:126:in `block (3 levels) in <top (required)>'
Tasks: TOP => redmine:plugins:migrate
(See full trace by running task with --trace)
```
I've fixed them and wish my pull request being merged (coming pretty soon)

**Comment by go2null** [comment]
Noticed that you are using MariaDB, not MySQL proper.  This may be the source of the issue.

**Comment by go2null** [comment]
Fixed by Anton in v0.0.5 (764134eeea0ede00d31e9ff58e0bd3dde4d28a78)
https://github.com/anteo/redmine_custom_workflows/commit/e05a8a467d4242d26070f30e17a77def50165bda
